### ISSUE TITLE
Bug report same images are multiply generated when using multiple workers.

### ISSUE THREAD

**Issue by akiraf0421** [uoguelph-mlrg/Cutout#17]
When I use torch.utils.data.DataLoader init with multi worker and also using Cutout for transform, I observed same transformed images were generated. This number of images are same as number of workers. I guess this is because use of numpy.random in Cutout source code. In torchvision.transforms, numpy seems to be initialized by the same seed in each workers.

This issue can be fixed by replacing numpy.random with torch.randint in Cutout source code.
